lib/database/models/typedefs/SequelizeQualityControlFlag.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef SequelizeQualityControlFlag
+ *
+ * @property {number} id
+ * @property {number} timeStart
+ * @property {number} timeEnd
+ * @property {string} comment
+ * @property {enum{'HUMAN', 'SYNC', 'ASYNC', 'MC'}} provenance
```

Comment:
I think the syntax is `{'item1'|'item2'}`

Reply:
applied